jenkins_jobs/modules/triggers.py

```diff
@@ -511,6 +552,19 @@ def gerrit(registry, xml_parent, data):
         to Gerrit. (default false)
     :arg bool escape-quotes: escape quotes in the values of Gerrit change
         parameters (default true)
+    :arg dict build-cancellation-policy: If used, only running jobs will be
+        cancelled if a new patch version is pushed over.
+        Requires Gerrit Trigger Plugin version >= 2.32.0
+
+        :Options: * **enabled** (`bool`)
+                  * **abort-new-patchsets** (`bool`) -- Only running jobs
+                    will be cancelled if a new patch version is pushed over.
+                  * **abort-manual-patchsets** (`bool`) -- Builds triggered
+                    manually will be aborted when a new patch set arrives.
+                  * **abort-same-topic** (`bool`) -- Builds triggered with
+                    topic will be aborted when a new patch set with the
+                    same topic arrives.
```

Comment:
If not specified, what the defaults are?

Reply:
Specified.